arch/arm/boot/dts/imx6qdl-ts4900-ts8950.dtsi

```diff
@@ -0,0 +1,339 @@
+// SPDX-License-Identifier: GPL-2.0 OR X11
+/*
+ * Copyright 2018-2022 Technologic Systems, Inc. dba embeddedTS
+ */
+
+#include <dt-bindings/interrupt-controller/irq.h>
+
+/ {
+	chosen {
+		stdout-path = &uart1;
+	};
+
+	aliases {
+		ethernet0 = &fec;
+	};
+
+	backlight_lvds: backlight-lvds {
+		compatible = "pwm-backlight";
+		pwms = <&pwm3 0 500000 0>;
+		/*
+		 * Backlight doesn't register anything until about half of max
+		 * brightness. So start the first real step at something that
+		 * makes the display visible. From there, make each step
+		 * meaningful along the scale.
+		 */
+		brightness-levels = < 0 48 49 50 51 52 53 54 55 56
+				     57 58 59 60 61 62 63 64 65 66
+				     67 68 69 70 71 72 73 74 75 76
+				     77 78 78 80 81 82 83 84 85 86
+				     87 88 89 90 91 92 93 94 95 96
+				     97 98 99 100>;
+		default-brightness-level = <53>;
```

Comment:
Shouldn't we default this to 100%?  This might be a surprise if its not on full by default.

Reply:
This is actually correct, albeit confusing. There are 54 brightness level values, 0, 48-100. With a 0 level being 0% brightness, and level 53 being 100%.

In my research and testing there was no real good way to say make 0 be 0%, and then the meaningful PWM values automatically scale from 1% to 100%. I'd be happy to rework this a bit if you have an idea that might make more sense?